config/routes.rb

```diff
@@ -3,12 +3,12 @@
   resources :homes, only: [:index]
```

Comment:
If we don't need this path lets remove line 3

Reply:
removed it